src/components/RadioGroup/RadioGroup.stories.mdx

```diff
@@ -0,0 +1,66 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Preview,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+import Radio from "../Radio/Radio";
+
+import { getCategory } from "../../utils/componentCategories";
+import RadioGroup, { RadioGroupLayouts } from "./RadioGroup";
+import { onChangeDefault } from "./RadioGroup";
+
+<Meta
+  title={getCategory("RadioGroup")}
+  component={RadioGroup}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=11895%3A742",
+    },
+    jest: ["RadioGroup.test.tsx"],
+  }}
+  argTypes={{
+    children: { table: { disable: true } },
+    onChange: { table: { disable: true } },
+    id: { table: { disable: true } },
+  }}
+/>
+
+# RadioGroup
+
+<Description of={RadioGroup} />
+
+<Preview withToolbar>
+  <Story
+    name="RadioGroup"
+    args={{
+      name: "radio-story",
+      defaultValue: "4",
+      layout: RadioGroupLayouts.Vertical,
+      labelText: "radio group label",
+      helperText: "this is the helper text",
+      errorText: "error!",
+      showLabel: true,
+      errored: false,
+      required: false,
+      disabled: false,
+    }}
+  >
+    {(args) => (
+      <RadioGroup {...args}>
+        <Radio value="2" labelText="Radio 2" showLabel />
```

Comment:
One reason why I want to work on `Radio` first is because the `showLabel` prop is required but it doesn't make sense to _not_ show it. You won't know the value you are selecting.

Reply:
Also, I think we should stop using the `labelText` prop and make it an explicit child `<Radio value="3">Radio 3</Radio>`

Reply:
We included `showLabel` for both `Radio` and `Checkbox` to allow for scenarios where the inputs were used to select a row of tabular data or an image.  In both cases, you wouldn't need a visible label, but you would want something that a screen readers could see.

Regarding `labelText`, to better align with Chakra, yes, I think it would be OK to have that come in as a child.  However, while we don't currently have any designs that require a `Radio` without a label, I think we should retain `showLabel`.  Unless there is a way to do this in the Chakra component that I am not seeing.

Reply:
Okay, let's keep `showLabel` but it should be set to "true" by default.

Reply:
Yes, agreed.  I am not sure why `false` was the default.